waffle-jest/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@ethereum-waffle/jest",
+  "description": "A sweet set of jest matchers for your blockchain testing needs.",
+  "version": "3.0.0",
+  "author": "Adrian Li <li.adrianmc@gmail.com>",
+  "repository": "git@github.com:EthWorks/Waffle.git",
+  "private": false,
+  "license": "MIT",
+  "keywords": [
+    "ethereum",
+    "smart-contracts",
+    "solidity",
+    "testing",
+    "javascript",
+    "typescript",
+    "jest",
+    "library"
+  ],
+  "homepage": "https://github.com/EthWorks/Waffle",
+  "bugs": {
+    "url": "https://github.com/EthWorks/Waffle/issues"
+  },
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.ts",
+  "types": "dist/esm/index.d.ts",
+  "scripts": {
+    "prepublishOnly": "yarn build",
+    "test": "jest",
+    "lint": "eslint '{src,test}/**/*.ts'",
+    "lint:fix": "eslint --fix '{src,test}/**/*.ts'",
+    "build": "rimraf ./dist && yarn build:esm && yarn build:cjs",
+    "build:esm": "tsc -p tsconfig.build.json --outDir dist/esm --module ES6",
+    "build:cjs": "tsc -p tsconfig.build.json --outDir dist/cjs --declaration false"
+  },
+  "engines": {
+    "node": ">=10.0"
+  },
+  "dependencies": {
+    "@ethereum-waffle/provider": "^3.0.0",
```

Comment:
Would be good to remove that dependency, to do so validation in `validateMockProvider` could be relaxed.